gcloud-java-cloudtrace/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>gcloud-java-cloudtrace</artifactId>
```

Comment:
Uh yes sure I meant `Stackdriver Trace`. 

My preference would be to call this just `gcloud-java-trace`, and call the package `com.google.cloud.trace.spi.v1`.